Scripts/validateMavenVersion.ps1

```diff
@@ -43,9 +43,17 @@ $bintrayAPIurl = "https://api.bintray.com/search/packages?name=$packageName"
 $jsonResult = $web_client.DownloadString($bintrayAPIurl) | ConvertFrom-Json
 $bintrayVersion = [version]$jsonResult.latest_version
 
+#If the api calls return empty then this library cannot be compared to the online versions
+#may proceed with the pull request
+if(($mavenVersion -eq $null) -and ($bintrayVersion -eq $null))
```

Comment:
I am aware that this PR is merging from `dev` to `master`, this can be addressed in another PR to `dev`, so it is not a blocking comment.

Can you please confirm if all of the following input/output pairs are expected?
```powershell
$localVersion = [version]"2.0.0"
$mavenVersion = [version]$null
$bintrayVersion = [version]"1.0.0"
```
```
The current version in the Maven central repository is:
The current version in the Bintray central repository is: 1.0.0
WARNING: The current Maven and Bintray versions are not the same
The current pull request is of a greater version
```
```powershell
$localVersion = [version]"2.0.0"
$mavenVersion = [version]"1.0.0"
$bintrayVersion = [version]$null
```
```
The current version in the Maven central repository is: 1.0.0
The current version in the Bintray central repository is:
WARNING: The current Maven and Bintray versions are not the same
The current pull request is of a greater version
```
```powershell
$localVersion = [version]"2.0.0"
$mavenVersion = [version]$null
$bintrayVersion = [version]"2.0.0"
```
```
The current version in the Maven central repository is:
The current version in the Bintray central repository is: 2.0.0
WARNING: The current Maven and Bintray versions are not the same
C:\github\msgraph-sdk-java-core\Scripts\validateMavenVersion.ps1 : The current local version is not updated. Please update the local version in the Gradle.Properties file.
At line:1 char:1
+ .\validateMavenVersion.ps1
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,validateMavenVersion.ps1
```
```powershell
$localVersion = [version]"2.0.0"
$mavenVersion = [version]"2.0.0"
$bintrayVersion = [version]$null
```
```
The current version in the Maven central repository is: 2.0.0
The current version in the Bintray central repository is:
WARNING: The current Maven and Bintray versions are not the same
The current pull request is of a greater version
```

Reply:
thanks for catching that! I was simply trying to align branches at this point.
I think the inconsistencies in pair value testing results came from the last condition. I authored a PR to fix it #54 
This current PR will be good once the other one gets merged.